langchain/src/vectorstores/googlevertexai.ts

```diff
@@ -0,0 +1,682 @@
+import * as uuid from "uuid";
+import flatten from "flat";
+import { VectorStore } from "./base.js";
+import { Embeddings } from "../embeddings/base.js";
+import { Document, DocumentInput } from "../document.js";
+import { GoogleVertexAIConnection } from "../util/googlevertexai-connection.js";
+import {
+  AsyncCaller,
+  AsyncCallerCallOptions,
+  AsyncCallerParams,
+} from "../util/async_caller.js";
+import {
+  GoogleVertexAIConnectionParams,
+  GoogleVertexAIResponse,
+} from "../types/googlevertexai-types.js";
+import { Docstore } from "../schema/index.js";
+
+export interface IdDocumentInput extends DocumentInput<Record<string, any>> {
+  id?: string;
+}
+
+export class IdDocument
+  extends Document<Record<string, any>>
```

Comment:
Can just do `extends Document` to make the linter happy

Reply:
No objections.